Profile page initial template

michael-caira opened the pull request:
@leandrabudau2939 and I, Implemented the profile page figma: https://www.figma.com/file/LLZiwB6naD3qtkoG6seG27/Image-Repo-UI?node-id=8%3A51

Everything is hardcoded as specified for now, but the template should be good to go for connecting to real data.

One thing we may need is a modal for uploading a new user profile pic, but we could just do something basic/not do a drag and drop type of modal.

Let us know what you think.

Mwpereira commented:
Currently, there is no functionality for this button?

![image](https://user-images.githubusercontent.com/56651345/200028509-1d2a600d-5efb-4e8a-9590-b0b498d8c1d5.png)

jamestang12 commented on client/src/App.tsx:
```diff
@@ -1,7 +1,7 @@
+import { BrowserRouter as Router, Navigate, Route, Routes } from 'react-router-dom';
 
 import Dashboard from './pages/Dashboard';
 // import Container from './components/Container';
```
Please remove this if we are not using it

jamestang12 reviewed (commented):
Overall LGTM, please remove unnecessary import and fix the styling issue mentioned by @Mwpereira 

Great work 👍

michael-caira commented:
Good catches, added functionality for image selection, removed unused imports, and added more spacing, let me know if you guys think it's good.

Tried to keep the spacing consistent with the spacing at the top between the user handle and the beginning of the form.
![Screenshot 2022-11-06 at 10 42 05 PM](https://user-images.githubusercontent.com/58964071/200222010-f9270ebb-5e72-466d-8516-6cf08f4e95d4.png)

jamestang12 reviewed (approved):
LGTM
